website_docs/getting-started.md

```diff
@@ -1,12 +1,13 @@
 ---
-date: '2021-05-07T21:49:47.106Z'
+date: '2021-08-30T16:49:17.700Z'
 docname: getting-started
 images: {}
 path: /getting-started
-title: "Getting Started"
-weight: 22
+title: Getting Started
 ---
 
+# Getting Started
```

Comment:
This change causes the title to be generated twice on the website. For a preview of this see: https://deploy-preview-805--opentelemetry.netlify.app/docs/python/getting-started/